hw/ip/otbn/doc/_index.md

```diff
@@ -427,6 +427,25 @@ Additionally, instructions and data stored in the instruction and data memory, r
 
 Refer to the [Data Integrity Protection]({{<relref "#design-details-data-integrity-protection">}}) section for details of how the data integrity protections are implemented.
 
+## Secure Wipe
+
+OTBN provides a mechanism to securely wipe all state it stores, including the instruction memory.
+
+The full secure wipe mechanism is split into three parts:
+- [Data memory secure wipe]({{<relref "#design-details-secure-wipe-dmem">}})
+- [Instruction memory secure wipe]({{<relref "#design-details-secure-wipe-imem">}})
+- [Internal state secure wipe]({{<relref "#design-details-secure-wipe-internal">}})
+
+A secure wipe is performed automatically in certain situations, or can be requested manually by the host software.
+The full secure wipe is automatically initiated as a local reaction to a fatal error.
+(TODO: Specify reaction to incoming lifecycle escalation event, and a potential lifecycle state dependency, e. g. only wipe in PROD mode.)
+A secure wipe of the internal state is performed whenever an OTBN operation is completed and after a recoverable error.
+Finally, host software can manually trigger the full or partial secure wipe operations by writing to the {{< regref "SEC_WIPE">}} register. 
+During the secure wipe operation the {{< regref "STATUS.busy">}} flag is set to 1, indicating that the OTBN is busy.
+Once the operation is completed, a {{< regref "INTR_STATE.done" >}} interrupt is set and {{< regref "STATUS.busy">}} is cleared. 
```

Comment:
The section this text appears in is the high-level description of security features. I would argue that the busy/interrupt behavior is better placed in the design details section further down.

And a nit:
```suggestion
Once the operation is completed, a {{< regref "INTR_STATE.done" >}} interrupt is raised and {{< regref "STATUS.busy">}} is cleared. 
```

On that topic: It is already implicitly defined, but maybe add one explicit sentence that OTBN being busy during the secure wipe operation, and writes to SEC_WIPE being ignored while OTBN is busy, means effectively that host software will get a single done interrupt for a secure wipe operation, independent of how many SEC_WIPE bits the software wrote.